components/kyma-environment-broker/internal/process/provisioning/cls_binding_step.go

```diff
@@ -124,8 +124,8 @@ func (s *ClsBindStep) Run(operation internal.ProvisioningOperation, log logrus.F
 		// fetch existing overrides
 		overrideParams, err = cls.DecryptOverrides(s.secretKey, operation.Cls.Overrides)
 		if err != nil {
-			failureReason := fmt.Sprintf("decryptClsOverrides() call failed")
-			log.Errorf("%s: %s", failureReason, err)
+			failureReason := fmt.Sprintf("Unable to decrypt CLS overrides")
```

Comment:
we don't need to use `Sprintf()`. Please fix for other occurrences without parameters
```suggestion
			failureReason := "Unable to decrypt CLS overrides"
```

Reply:
Fixed